app/models/chat_channel.rb

```diff
@@ -65,66 +80,23 @@ def url
     "#{Discourse.base_url}/chat/channel/#{self.id}/-"
   end
 
-  def chatable_url
-    return nil if direct_message_channel?
-
-    chatable.url
-  end
-
-  def category_channel?
-    chatable_type == "Category"
-  end
-
-  def direct_message_channel?
-    chatable_type == "DirectMessageChannel"
-  end
-
-  def public_channel?
-    ChatChannel.public_channel_chatable_types.include?(chatable_type)
-  end
-
-  def chatable_has_custom_fields?
-    category_channel?
-  end
-
-  def allowed_user_ids
-    direct_message_channel? ? chatable.user_ids : nil
-  end
-
-  def allowed_group_ids
-    return if !category_channel?
-    return if category_channel? && !read_restricted?
-
-    staff_groups = Group::AUTO_GROUPS.slice(:staff, :moderators, :admins).values
-    chatable.secure_group_ids.to_a.concat(staff_groups)
-  end
-
   def public_channel_title
     chatable.name
   end
 
-  def read_restricted?
-    return true if direct_message_channel?
-    return chatable.read_restricted? if category_channel?
-
-    true
-  end
-
-  def title(user)
-    return chatable.chat_channel_title_for_user(self, user) if direct_message_channel?
-    return name if name.present?
-
-    title_from_chatable
+  def category_channel?
+    false
   end
+  alias direct_message_channel? category_channel?
```

Comment:
I am confused by some of these aliases, they don't seem to line up to me, maybe I am reading it wrong? How is `category_channel?` an alias for `direct_message_channel?`

Reply:
It's mostly for defualt state, but we will try to make something simpler/more obvious.